### DISCUSSION TITLE
new repo for CTF?

### DISCUSSION THREAD

**Discussion by dangnguyencse** [dangnguyenphuc/cryptohack#4] [Q&A] [answered]
new repo for CTF?

**Comment by dangnguyenphuc** [comment, marked as answer]
yeah

**Comment by dangnguyenphuc** [comment]
https://drive.google.com/drive/folders/1F_7sMYYA2-IdqgYapqK7XUUJp6whQOiv?usp=sharing

**Reply by dangnguyenphuc** [reply]
https://drive.google.com/drive/folders/1j7ThM7RRbG0WqPz84R0o2fj4-Z_aJ-5O?usp=sharing

**Comment by dangnguyenphuc** [comment]
https://docs.google.com/document/d/1RYJ8bsE74uoQbyVcQu41fBXQbPQ3VP4A/edit?usp=sharing&ouid=100904496167172299165&rtpof=true&sd=true

**Comment by dangnguyenphuc** [comment]
https://www.pgs.hcmut.edu.vn/vi/lien-thong-dh-ths/2020-07-31-07-54-16/danh-muc-mon-hoc-xet-mien

**Comment by dangnguyenphuc** [comment]
SPE: https://drive.google.com/drive/folders/1ru_QYGVfpRaOPOoq2Do-85DcPVgEyK-R?usp=sharing

**Comment by dangnguyenphuc** [comment]
https://drive.google.com/drive/folders/1xOQF0JPaDZ8sKXey1-V0TbVbylCFG7-C?usp=sharing
DTB

**Comment by dangnguyenphuc** [comment]
This is Thread for thesis report

**Reply by dangnguyenphuc** [reply]
Mission Eval: https://docs.google.com/document/d/15p0DltxAG6c3VeXgJHA4NBs3aOaA-qYV/edit?usp=sharing&ouid=112084303802233024501&rtpof=true&sd=true

**Reply by dangnguyenphuc** [reply]
Report: https://drive.google.com/file/d/18hZGv7zsi8oBlehYS5BPRu0xSbnm5Bvx/view?usp=sharing

**Comment by dangnguyenphuc** [comment]
Thank you for giving me the opportunity to introduce myself.

My name is Nguyen Phuc Dang, and im currently 4th year student at HCMUT. I had done my thesis in June and gonna graduate in November with bachelor degree in Computer Enginerring.

During my studies, i had did several game project using c, c++ that helped me to build my solid foundation in game development. For example, my project about platform game, this pro-
ject involve implementing collision, physics, implementing tilemap and handling footballer behaviors using c++ and SDL libraries.

To improve my skills and my knowledge, i had taken multiple courses about IoT, Cryptography, Cloud services etc

and that's all

**Comment by dangnguyenphuc** [comment]
I am eager to apply for this position because it perfectly aligns with my career goals and aspirations. My objective is to further my career in game development by contributing my skills and passion to a team focused on creating immersive and innovative gaming experiences.

The connection between my passion and this position is profound. I have always been fascinated by the intersection of creativity and technology, particularly in the realm of game design and development. This role represents an ideal opportunity for me to combine my technical proficiency in programming with my creative drive to contribute meaningfully to game projects.

I am particularly enthusiastic about the responsibilities associated with this position. I thrive in environments that challenge me to innovate and problem-solve, and I am committed to delivering high-quality work that exceeds expectations. My dedication to continuous learning and improvement ensures that I approach every project with enthusiasm and a willingness to grow.

This position offers me a platform to advance both technically and professionally. I see it as a chance to deepen my expertise in game development, learn from experienced colleagues, and contribute to the success of projects that inspire and entertain players worldwide. I am confident that my skills, combined with my passion and commitment, make me a strong fit for this role and a valuable asset to your team.

**Comment by dangnguyenphuc** [comment]
![networkStat](https://github.com/user-attachments/assets/af07800e-5a25-4c37-94d7-237f5c76e03e)
